tests/linkplay/test_bridge.py

```diff
@@ -212,6 +212,18 @@ async def test_player_set_play_mode():
     )
 
 
+async def test_player_play_preset():
```

Comment:
Could you parameterize this in two functions for both valid and invalid data? Maybe something like this. I'll also add this to the volume test.

```python
@pytest.mark.parametrize("preset_number", range(1, 11))
async def test_player_play_preset(preset_number: int):
    bridge = AsyncMock()
    player = LinkPlayPlayer(bridge)

    await player.play_preset(preset_number)
    
    bridge.request.assert_called_once_with(
      LinkPlayCommand.PLAY_PRESET.format(preset_number)
    )


@pytest.mark.parametrize("preset_number", [
    0,
    11,
    "abc",
    None,
])        
async def test_player_play_preset_raises_value_error(preset_number: Any):
    bridge = AsyncMock()
    player = LinkPlayPlayer(bridge)
    
    with pytest.raises(ValueError):
        await player.play_preset(preset_number)
  
```

Reply:
Done.

Note I wasn't able to add `"abc"` and `None` tests as it was failing with a different error (not super familiar with python's typing constructs, hopefully the test is ok as it stands?).  Let me know if you have further enhancement ideas :)